overlord/auth/auth.go

```diff
@@ -206,6 +208,8 @@ NextUser:
 
 // An AuthContext handles user updates.
 type AuthContext interface {
```

Comment:
Let's please not overdo documentation. Device returns a device, and UpdateDevice updates a device. There's really no need to document these methods individually here.